openshift-client/src/test/java/io/fabric8/openshift/client/Handlersv4Test.java

```diff
@@ -0,0 +1,66 @@
+package io.fabric8.openshift.client;
+
```

Comment:
Could you please add a license header here? This command would add it automatically:
```
mvn -N license:format
```